src/neo/UInt160.cs

```diff
@@ -1,22 +1,28 @@
+using Neo.IO;
 using System;
 using System.Globalization;
 using System.IO;
+using System.Runtime.InteropServices;
 
 namespace Neo
 {
     /// <summary>
     /// This class stores a 160 bit unsigned int, represented as a 20-byte little-endian byte array
     /// </summary>
-    public class UInt160 : UIntBase, IComparable<UInt160>, IEquatable<UInt160>
+    [StructLayout(LayoutKind.Explicit, Size = 20)]
+    public class UInt160 : IComparable<UInt160>, IEquatable<UInt160>, ISerializable
     {
         public const int Length = 20;
         public static readonly UInt160 Zero = new UInt160();
 
-        private ulong value1;
-        private ulong value2;
-        private uint value3;
+        /// <summary>
+        /// ulong(64) + ulong(64) + uint(32) = UInt160(160)
+        /// </summary>
```

Comment:
These xml comments are for `value1` only. And also, I think we don't need these comments.

Reply:
Moved up to the class.